ResearchKit/Localized/en.lproj/ResearchKit.strings

```diff
@@ -340,10 +340,15 @@
 "HOLE_PEG_TEST_TEXT_2" = "Lift fingers to drop the peg.";
 
 /* Tremor test active task. */
-"TREMOR_TEST_TITLE" = "Tremor Test";
+"TREMOR_TEST_TITLE" = "Tremor Activity";
 "TREMOR_TEST_INTENDED_USE" = "This activity measures the tremor of your hands in various positions. Perform this test with your more affected hand.";
 "TREMOR_TEST_INTRO_2_TEXT" = "Hold the phone as shown in the image below and find a place where you can sit comfortably for the duration of this task.";
-"TREMOR_TEST_INTRO_2_DETAIL" = "You will be asked to perform four tasks while in this position.";
+"TREMOR_TEST_INTRO_2_DETAIL_%" = "You will be asked to perform %@ while in this position.";
```

Comment:
Might be worth some back-and-forth with Christine on the wording? I don't think my 6 year old would understand those instructions.

"You will be asked to perform %@ while sitting with the phone in your hand."